src/AspNetCoreModuleV2/RequestHandler/dllmain.cxx

```diff
@@ -303,7 +303,7 @@ CreateApplication(
             goto Finished;
         }
 
-        pApplication = new OUT_OF_PROCESS_APPLICATION(pServer, pConfig);
```

Comment:
we passed pServer to out-of-process application and in-process application as we want to have the ability to read the configuration inside the class instead of using the passed the configuration instance., e.g., requesthandler  needs to read configuration setting from handlerSettings

Reply:
We can still do it, but there is no point in keeping unused parameter around until then